### PR TITLE
Add e2e test that kills ovnkube-master; fix ovnkube-node test for multiple nodes

### PR DESCRIPTION
**Pull request by as-com**
This PR adds a new e2e test that ensures Internet connectivity is maintained if the ovnkube-master pod is killed. I also fixed the existing ovnkube-node test to handle multiple nodes now that it's a thing in the KIND test suite.

### CODE REVIEW

File: test/e2e/e2e_test.go [hunk 1]
```diff
@@ -99,7 +111,7 @@ var _ = Describe("e2e control plane", func() {
 		podList, _ := podClient.List(metav1.ListOptions{})
 		podName := ""
 		for _, pod := range podList.Items {
-			if strings.HasPrefix(pod.Name, "ovnkube-node") {
+			if strings.HasPrefix(pod.Name, "ovnkube-node") && pod.Spec.NodeName == testPod.Spec.NodeName {
```
Comment (danwinship) [inline test/e2e/e2e_test.go:114]: ideally this PR would be two separate commits, one fixing the existing test and one adding a new test

File: test/e2e/e2e_test.go [hunk 2]
```diff
@@ -111,4 +123,33 @@ var _ = Describe("e2e control plane", func() {
...
+	ginkgo.It("should provide Internet connection continuously when master is killed", func() {
+		ginkgo.By("Running container which tries to connect to 8.8.8.8 in a loop")
+
+		podChan, errChan := make(chan *v1.Pod), make(chan error)
+		go checkContinuousConnectivity(f, "", "connectivity-test-continuous", "8.8.8.8", 53, 30, podChan, errChan)
+
+		testPod := <-podChan
+		framework.Logf("Test pod running on %q, waiting a few seconds", testPod.Spec.NodeName)
+
+		time.Sleep(10 * time.Second)
```
Comment (danwinship) [inline test/e2e/e2e_test.go:136]: What is this for? There are helper routines in the kubernetes e2e framework for, eg, waiting until a pod reports a Ready status, etc. That's faster when the cluster is fast and reliable when the cluster is slow.

Comment (as-com) [reply test/e2e/e2e_test.go]: I just removed the wait, since the checkContinuousConnectivity function now waits until the pod exit the pending state before allowing the test to continue.
